Source/ARTClientOptions.m

```diff
@@ -76,7 +75,7 @@ - (NSString*)getRealtimeHost {
     if ([_environment isEqualToString:ARTDefaultProduction]) {
         return [ARTDefault realtimeHost];
     }
-    return _environment ? [NSString stringWithFormat:@"%@-%@", _environment, [ARTDefault realtimeHost]] : [ARTDefault realtimeHost];
+    return self.hasEnvironment ? [NSString stringWithFormat:@"%@-%@", _environment, [ARTDefault realtimeHost]] : [ARTDefault realtimeHost];
```

Comment:
Consider encapsulating this [NSString stringWithFormat:@"%@-%@", _environment, [ARTDefault restHost]] as a Helper/Util/Category